### ISSUE TITLE
Fine-tune pi0-fast

### ISSUE THREAD

**Issue by JING-SHAO** [ai4ce/INT-ACT#6]
Hi, have you fine-tuned pi0-fast using this code? I tried it and found that at the 2500th batch, the loss was quite high, around 5.7. Have you encountered this issue before?

<img width="914" height="176" alt="Image" src="https://github.com/user-attachments/assets/08cf2d59-f5b2-4211-96ad-f1a562540404" />

**Comment by IrvingF7** [comment]
Hi!
Thank you for your interest in our work.

We tried to train pi0-fast using this codebase, but at the time of our experiment, LeRobot's pi0-fast implementation was buggy, according to their maintainers, so we were not successful. We tried to compare Physical Intelligence's JAX implementation, but to no avail.

Hope this answers your question!
